data_collection/gazette/spiders/sp_americana.py

```diff
@@ -0,0 +1,68 @@
+import dateparser
+from scrapy import Spider
+import datetime
+from dateutil.relativedelta import relativedelta
+from gazette.items import Gazette
+import re
+
+
+class SpAmericanaSpider(Spider):
+    TERRITORY_ID = "3501608"
+    name = "sp_americana"
+    allowed_domains = ["diariooficial.americana.sp.gov.br"]
+    start_date = datetime.date(2018, 9, 1)  # First gazette available
+    start_urls = ["https://diariooficial.americana.sp.gov.br/diario-oficial-edicaoAnterior.php"]
+    extra_editions = "https://diariooficial.americana.sp.gov.br/diario-oficial-edicaoExtra.php"
+
+    locations = {
+        "gazette": "//*[@class='day table-light']/strong/a",
```

Comment:
**suggestion** When searching elements by class, prefer the use of CSS selectors instead of XPath. Any change in the class definition may break this selector. For example, if we have `<td class="day table-light">` and for some reason it changes to `<td class="table-light day">`, your selector will break even if nothing else changes.

Reply:
Using this construction it's like contains at XPath, too interesting!